.env.dist

```diff
@@ -2,3 +2,5 @@
 # please change your private key.
 MINER_KEY='EE006BF8B7802ED2443DC8B8F9A8A4A5B9F760713ED7C0AC21CB59C91C7C43A5'
 MINER_SLEEP_SEC=0
+MIN_BLOCK_INTERVAL = 5
```

Comment:
Sorry for late find. 

Bash shell can't parse this form (`A = B`) properly, I think you should rewrite like `MIN_BLOCK_INTERVAL=5`.